### PR TITLE
add Prometheus and Grafana roles (SES-199)

### PR DESCRIPTION
**Pull request by jan--f**
and install and configure accordingly
[SES-199](https://jira.suse.de/browse/SES-199)

### CODE REVIEW

**Comment by jschmid1** [comment]
yummy 6,5k lines deleted :sake:

**Comment by capri1989** [comment]
Yeah we can configure Grafana and Prometheus individually. Thanks a lot!

**Comment by jschmid1** [comment]
Since we have two new roles, we should also add them to `cephprocesses` and add a separate restart logic, right?

**Comment by jan--f** [comment]
> Since we have two new roles, we should also add them to `cephprocesses` and add a separate restart logic, right?

Huh I though I had added them to cephprocesses already. Coming up. Restarts are trivial, things can just be restarted whenever.

File: srv/salt/ceph/stage/deploy/default.sls [hunk 1]
```diff
@@ -1,5 +1,32 @@
+{% set master = salt['master.minion']() %}
+
 include:
   - .core
   - ...restart.mon.lax
   - ...restart.mgr.lax
   - ...restart.osd.lax
+
+enable prometheus module:
```
Comment (jschmid1) [inline srv/salt/ceph/stage/deploy/default.sls]: Maybe we should encapsulate these steps like we did with 'core'
```
.core
...restart.mon.lax
...restart.mon.lax
...restart.mon.lax
.monitoring_deploy
```

also, if we have the restart logic in place, we should add it similar to the ...restart.$role.lax approach

Comment (jan--f) [reply srv/salt/ceph/stage/deploy/default.sls]: hmm point taken. Unfortunately there is more to the monitoring deployment (stage 2 has the majority of it).

**Comment by kshtsk** [comment]
retest this please

**Comment by jan--f** [comment]
Closing in order to test JIRA integration

**Pull request closed without merging**